### ISSUE TITLE
'timeInForce' sent when not required. in PlaceOrderAsync

### ISSUE THREAD

**Issue by heavymanto** [JKorf/Binance.Net#705]
Hi, i have test to pace an order on Binance client with PlaceOrderAsync method but it respond aways "'timeInForce' sent when not required.". I have funds in both accounts.

CODE
```
  string _symbol = ((IExchangeClient)_client).GetSymbolName(_valutaBase, _valutaQuote);
  IExchangeClient.OrderSide _operation = (IExchangeClient.OrderSide)(int)operation;

  var order = ((IExchangeClient)_client).PlaceOrderAsync(_symbol, _operation, IExchangeClient.OrderType.Market, amount);
 ```

DATA/LOG OF ORDER RESULT
debugData --> **symbol**: TRXBTC, **operation**: Buy, **amount**: 480,815463
ApiClient_Binance.PlaceOrder --> BadRequest,Parameter 'timeInForce' sent when not required.

**Comment by heavymanto** [comment]
I found bug in class BinanceClient IExchangeClient.PlaceOrderAsync 

![image](https://user-images.githubusercontent.com/6449169/116085931-9a14d700-a69f-11eb-9cb9-e8450807040b.png)

timeInForce is mandatory only if  OrderType = Limit, on Market Exchange raise exception

**Comment by JKorf** [comment]
Hi you're right, I've fixed it in version 6.13.3.
